src/components/Footer.jsx

```diff
@@ -1,7 +1,46 @@
+import img1 from "@/public/footer.webp";
+import wavetop from "@/public/waveTop.webp";
+import { footerLinks } from "../data/footerLinks";
+import Link from "next/link";
+import Image from "next/image";
+
 const Footer = () => {
   return (
-    <div className="">
-      <p>Footer</p>
+    <div className="flex flex-col relative w-page h-auto">
+      <div className="flex w-screen mt-28">
+        <Image alt="img1" src={img1} className="w-screen h-full" />
+      </div>
+      <div className="flex w-screen absolute top-0 pb-[37.2%] bg-gradient-to-b from-white/90 to-swim-blue-300/30 bg-origin-padding">
+        <Image alt="wavetop" src={wavetop} className="w-screen h-full" />
+      </div>
+      <div className="flex justify-between text-white">
+        <Link href="/" className="hover:opacity-80">
+          <div className="text-5xl font-bold pl-10 pb-32 self-end justify-self-start absolute bottom-0 left-0">
```

Comment:
```suggestion
          <div className="text-5xl font-bold ml-10 mb-32 self-end justify-self-start absolute bottom-0 left-0">
```

use margining in this case. this is because padding adds space inside of the divider, so the link will highlight on empty spaces.